include/arch/arm/cortex_m/mpu/arm_mpu.h

```diff
@@ -44,6 +44,9 @@ struct arm_mpu {
 #define ARM_MPU_PRIVDEFENA	(1 << 2)
 
 #define REGION_VALID	(1 << 4)
+/* ARM MPU RABR Register */
```

Comment:
> The name of the MPU register is RBAR. Please check even the commit message.
Corrected.

I also modified the commit message.
Please help review them again. Thanks.